cfg/config.yaml

```diff
@@ -25,6 +25,7 @@ master:
       - "hyperkube apiserver"
       - "hyperkube kube-apiserver"
       - "apiserver"
+      - "openshift start master api"
```

Comment:
I was looking at OKD 3.11 and I noticed `hypershift openshift-kube-apiserver` instead of the above.